importer/helpers/helpers.go

```diff
@@ -38,16 +38,35 @@ var ErrSizeLimitExceeded = fmt.Errorf("object size limit exceeded")
 // of unixfs DAG trees
 type UnixfsNode struct {
 	node *dag.Node
-	ufmt *ft.MultiBlock
+	ufmt *ft.FSNode
 }
 
 func NewUnixfsNode() *UnixfsNode {
 	return &UnixfsNode{
 		node: new(dag.Node),
-		ufmt: new(ft.MultiBlock),
+		ufmt: &ft.FSNode{Type: ft.TFile},
 	}
 }
 
+func NewUnixfsBlock() *UnixfsNode {
+	return &UnixfsNode{
+		node: new(dag.Node),
+		ufmt: &ft.FSNode{Type: ft.TRaw},
```

Comment:
well... we havent ever been able to do that. We've gotten lucky that we havent run into a situation where its bitten us.